src/validation.h

```diff
@@ -175,7 +177,10 @@ extern bool fCheckBlockIndex;
 extern bool fCheckpointsEnabled;
 extern size_t nCoinCacheUsage;
 /** A fee rate smaller than this is considered zero fee (for relaying, mining and transaction creation) */
-extern CFeeRate minRelayTxFee;
+//mlumin 5/2021: changing variable name to Rate vs Fee because thats what it is.
+extern CFeeRate minRelayTxFeeRate;
+//mlumin 5/2021: add minWalletTxFee to separate out wallet and relay.
+extern CFeeRate minWalletTxFeeRate;
```

Comment:
needs to move to `wallet.h` consistent with moving the value assignment to `wallet.cpp`.

To save further review back-and-forth, note that this also means that in `dogecoin-fees.cpp` a `#ifdef ENABLE_WALLET`-wrapped include of `wallet.h` is needed, which is consistent with everything else we're doing re: the wallet fees in that file.